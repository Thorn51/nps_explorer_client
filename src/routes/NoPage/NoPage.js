import React from "react";
import Navigation from "../../components/Navigation/Navigation";
import Hero from "../../components/Hero/Hero";
import "./NoPage.css";

export default function NoPage() {
  return (
    <>
      <Navigation />
      <Hero title="National Parks Explorer" />
    </>
  );
}
