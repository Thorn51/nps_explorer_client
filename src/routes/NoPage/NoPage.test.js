import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import NoPage from "./NoPage";
import { BrowserRouter } from "react-router-dom";

describe("<NoPage />", () => {
  it("Renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <NoPage />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });

  it("Renders the UI as expected", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <NoPage />
        </BrowserRouter>
      )
      .toJSON();
    expect("tree").toMatchSnapshot();
  });
});
