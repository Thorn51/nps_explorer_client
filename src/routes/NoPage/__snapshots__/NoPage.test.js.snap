// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoPage /> Renders the UI as expected 1`] = `"tree"`;
